.github/workflows/pipeline.yml

```diff
@@ -99,6 +99,8 @@ jobs:
             ${{ runner.os }}-docker-
 
       - name: Build and cache image from Dockerfile
+        env:
+          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
```

Comment:
The pipeline requires access to this token since it uploads the sourcemaps to sentry so that you have access to stack traces: https://docs.sentry.io/platforms/javascript/sourcemaps/